pwiz_tools/Shared/ProteowizardWrapper/MsDataFileImpl.cs

```diff
@@ -145,7 +161,7 @@ public static bool IsSingleIonCurrentId(string id)
                     ignoreZeroIntensityPoints = ignoreZeroIntensityPoints,
                     preferOnlyMsLevel = preferOnlyMsLevel,
                     allowMsMsWithoutPrecursor = false,
-                    //isolationMzAndMobilityFilter = new List<MzMobilityWindow> { new MzMobilityWindow(895.9496, 1.12, 0.055)}
+                    isolationMzAndMobilityFilter = precursorMzAndIonMobilityWindows?.Select(w => new MzMobilityWindow(w.MZ, w.IonMobility??0 , (w.IonMobilityWindow??0)/2)).ToList()

```

Comment:
Don't pass 0 here, pass null or use the 1 argument constructor.

Reply:
Roger that, thanks. Pushed.